modules/cnf-topology-aware-lifecycle-manager-operator-update.adoc

```diff
@@ -0,0 +1,262 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="talo-operator-update_{context}"]
+= Performing an Operator update
+
+You can perform an Operator update with the {cgu-operator}.
+
+.Prerequisites
+
+* Install the {cgu-operator-first}.
+* Update ZTP to the latest version.
+* Provision one or more managed clusters with ZTP.
+* Mirror the desired index image, bundle images, and all Operator images referenced in the bundle images.
+* Log in as a user with `cluster-admin` privileges.
+* Create {rh-rhacm} policies in the hub cluster.
+
+.Procedure
+
+. Update the `PolicyGenTemplate` CR for the Operator update.
+.. Update the `du-upgrade` `PolicyGenTemplate` CR with the following additional contents in the `du-upgrade.yaml` file:
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  name: "du-upgrade"
+  namespace: "ztp-group-du-sno"
+spec:
+  bindingRules:
+    group-du-sno: ""
+  mcp: "master"
+  remediationAction: inform
+  sourceFiles:
+    - fileName: DefaultCatsrc.yaml
+      remediationAction: inform
+      policyName: "operator-catsrc-policy"
+      metadata:
+        name: redhat-operators
+      spec:
+        displayName: Red Hat Operators Catalog
+        image: registry.example.com:5000/olm/redhat-operators:v4.10 <1>
+        updateStrategy: <2>
+          registryPoll:
+            interval: 10m
+----
+<1> The index image URL contains the desired Operator images. If the index images are always pushed to the same image name and tag, this change is not needed.
+<2> Set how frequently Operator Lifecycle Manager (OLM) pulls the index image for new Operator versions with the `registrypoll.interval` field. This change is not needed if a new index image tag is always pushed for y-stream and z-stream Operator updates.
```

Comment:
It would be useful to point out to the reader that the polling `interval` can be set to a shorter value to expedite the upgrade, however, this would come at the cost of an increased computational load. The user can restore the polling `interval` to the default value (`1h`) upon successful completion of the upgrade to counteract this.